### PR TITLE
chore: Add expected review counts to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by jszwedko**
Signed-off-by: Jesse Szwedko <jesse@szwedko.me>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -152,8 +152,17 @@ docs: fix typos
 
 #### Reviews & Approvals
 
-All pull requests must be reviewed and approved by at least one Vector team
-member. The review process is outlined in the [Review guide](REVIEWING.md).
+All pull requests should be reviewed by:
+
+* No review required for cosmetic changes like whitespace, types, and spelling
```
Comment (lukesteensen) [inline CONTRIBUTING.md:157]: Should `types` be `typos`? :smile:

Comment (jszwedko) [reply CONTRIBUTING.md:157]: Hah. Oops. :smile:

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -152,8 +152,17 @@ docs: fix typos
 
 #### Reviews & Approvals
 
-All pull requests must be reviewed and approved by at least one Vector team
-member. The review process is outlined in the [Review guide](REVIEWING.md).
+All pull requests should be reviewed by:
+
+- No review required for cosmetic changes like whitespace, typos, and spelling
```
Comment (Hoverbear) [inline CONTRIBUTING.md:157]: This is technically not correct since we do need to review contributor trivial changes.

Comment (jszwedko) [reply CONTRIBUTING.md:157]: :+1: 10e50b14

**Review by Hoverbear — APPROVED** [review]

**Comment by jszwedko** [comment]
@bruceg :+1: I had included a note to that effect:

https://github.com/timberio/vector/pull/3666/files#diff-6a3371457528722a734f3c51d9238c13R163-R164

Is that what you mean?

Speaking of which, do you mind taking a peak @jamtur01 ?

**Review by jamtur01 — APPROVED** [review]
LGTM

**Comment by bruceg** [comment]
Sorry, reading comprehension fail again.

**Comment by jszwedko** [comment]
> Sorry, reading comprehension fail again. Yes, that's exactly what I meant (with the possible quibble of something stronger than "should").

:+1: I like "should" as it gives the author some discretion. They could decide, for example, to not wait for a CODEOWNER that is on vacation. If it becomes an issue, I'm happy to change it to `must`; I just figured these were more like "guidelines" rather than strict requirements.

**Pull request merged**